tools/functions

```diff
@@ -94,7 +94,7 @@ get_user_confirmation() {
 
 # Get version from the swagger file
 function get_swagger_version {
-    local file=${1:-"$FC_ROOT_DIR/src/firecracker/swagger/firecracker.yaml"}
+    local file=${1:-"$FC_ROOT_DIR/src/firecracker/openapi/firecracker.yaml"}
     grep -oP 'version: \K.*' "$file"
```

Comment:
Based on the build failure, this checks needs to be adapted to `grep -oP '^\s+version: \K.* "$file"`